### PR TITLE
feat(onboarding)[WIP]: build screen for follow artists reward path

### PR DESCRIPTION
**Pull request by TMcMeans**
The type of this PR is: _feat_

This PR solves [GRO-1096]

### Description

The intention of this code is to build out the Follow Artists rewards screen that occurs at the end of the Onboarding workflow. Currently, an `orderedSet` is being used to backfill featured artists and a search filter will be implemented. 

Acceptance criteria:

- [x] User sees follow artists screen
- [x] User sees artwork image
- [x] User sees default list of artists from an orderedSet in admin
- [x] User can search artists
- [x] User sees artists name populating after search
- [x] User can continue after selecting at least one artist 
- [x] User can follow as many artists as they would like 
- [x] User can click x and leave the onboarding flow 
- [ ] Responsiveness

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[GRO-1096]: https://artsyproduct.atlassian.net/browse/GRO-1096?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/Apps/Onboarding/Components/OnboardingOrderedSet.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+// import { extractNodes } from "v2/Utils/extractNodes"
+
+interface OnboardingOrderedSetProps {
+  orderedSet: OnboardingOrderedSet_orderedSet
+}
+
+export const OnboardingOrderedSet: FC<OnboardingOrderedSetProps> = ({
+  orderedSet,
+}) => {
+  // const artists = extractNodes(orderedSet)
```
Comment (dzucconi) [inline src/v2/Apps/Onboarding/Components/OnboardingOrderedSet.tsx:18]: You need to extract it from the connection field.

File: src/v2/Apps/Onboarding/Components/OnboardingOrderedSet.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+              ... on Artist {
+                name
+                slug
+                internalID
+                image {
+                  imageURL
+                  internalID
+                }
+                formattedNationalityAndBirthday
+              }
```
Comment (dzucconi) [inline src/v2/Apps/Onboarding/Components/OnboardingOrderedSet.tsx:66]: Use the fragment

Comment (TMcMeans) [reply src/v2/Apps/Onboarding/Components/OnboardingOrderedSet.tsx:66]: right... gotta spread those fields into the query so they're available on the Artist obj right?

Comment (dzucconi) [reply src/v2/Apps/Onboarding/Components/OnboardingOrderedSet.tsx:66]: Yep: to use any fragment container you need to include that specific associated fragment

File: src/v2/Apps/Onboarding/config.ts [hunk 3]
```diff
@@ -11,6 +11,8 @@ export const useConfig = ({ basis, onDone }: UseConfig) => {
   const workflowEngine = useRef(
     new WorkflowEngine({
       workflow: [
+        //TODO: remove follow_artists from workflow
+        VIEW_FOLLOW_ARTISTS,
```
Comment (dzucconi) [inline src/v2/Apps/Onboarding/config.ts:15]: If you remove this we can merge

Comment (dzucconi) [reply src/v2/Apps/Onboarding/config.ts:15]: (In general I'd just avoid committing stuff like this)

**Pull request merged**